docs/Researcher/cli-reference/runai-port-forwarding.md

```diff
@@ -0,0 +1,70 @@
+# runai port-forward
+
+## Description
+
+Forward one or more local ports to the selected job. The forwarding session ends when the selected job terminates.
+
+Usage:
+
+To use port forwarding:
+
+1. Open a local terminal session.
+2. Find the target server using `<runai list -p <name of project>>`.
+3. Run the following command:
+     `runai port-forward <JOB_NAME> [flags]`
+
+!!!Note
```

Comment:
I think we should remove this section (1 through 4)

Reply:
I can remove it, but do the users know how to use the CLI otherwise? If not then these instructions are needed.

Reply:
the thing is that these instructions are inaccurate

Reply:
Agreed, if the instructions are not correct then they need to be changed. But we still can't leave the users with no istructions. Feel free to suggested new changes to the instructions.

Reply:
Changed the instructions and had it reference the installation page.

Reply:
I meant removed the e lines below (with the SSH thing)